core/dbt/adapters/README.md

```diff
@@ -1 +1,30 @@
 # Adapters README
+
+The Adapters module is responsible for defining database connection methods, caching information from databases, how relations are defined, and the two major types of connection types we have - base and sql.
+
+# Directories
+
+## `base`
+
+Defines the most base implementation Adapters can use to build out full functionality sweet
```

Comment:
Remove "most".  Remove "sweet".

Reply:
will do pretty sure i brain lagged and meant "suite"